classes/ActionScheduler_wpPostStore.php

```diff
@@ -93,7 +93,7 @@ protected function save_action_group( $post_id, $group ) {
 
 	public function fetch_action( $action_id ) {
 		$post = $this->get_post( $action_id );
-		if ( empty($post) || $post->post_type != self::POST_TYPE || $post->post_status == 'trash' ) {
+		if ( empty($post) || $post->post_type != self::POST_TYPE ) {
```

Comment:
Since this line is already being modified, the `empty($post)` statement should be formatted for WP standards

Reply:
@JPry my personal preference is generally to maintain internal consistency rather than to apply progressive enhancement. That way, when you're working with a code base, you're not being presented with multiple competing standards. For me, that avoids interrupting my thinking on the more important parts of the code.

I'm OK with doing a widespread standards update on AS though (unlike WP core). It's not really something I feel the need to do, but if you want to do it, I'll happily review/merge the PR.

Reply:
> For me, that avoids interrupting my thinking on the more important parts of the code.

As far as I'm concerned, this is a completely valid reason to avoid making these changes now and leave it as a separate PR like you mentioned. For me, making these kinds of changes as I go is trivial and doesn't interrupt my flow, since I can highlight any relevant lines and use a single keyboard shortcut to reformat according to WP standards. But as I said in another comment, I understand that's not the same for everyone.